src/themes/default/components/checkbox.st.css

```diff
@@ -0,0 +1,51 @@
+:import {
+	-st-from: "../../../components/checkbox/checkbox.st.css";
+	-st-default: CB;
+}
+
+:import {
+    -st-from: "../variables.st.css";
+    -st-named:  color_MainText_Background,
+                color_Default,
+                color_Disabled2,
+                color_Outline,
+                borderDefault,
+                iconTick,
+                iconTickDisabled,
+                iconIntermediate,
+                iconIntermediateDisabled;
+}
+
+CB::box {
+    background: value(color_MainText_Background);
+    border: value(borderDefault);
+}
+
+CB:checked::box,
+CB:indeterminate::box {
+    background: value(color_Default);
+}
+
+CB:disabled::box {
+    background: value(color_Disabled2);
+}
+
+CB:focus::box {
+    outline: 4px solid value(color_Outline);
```

Comment:
The outline color seems lighter that what is defined in the design

Reply:
maybe, in default theme it is not relevant, we synced all colors.